### ISSUE TITLE
Нужна ли полная версия?

### ISSUE THREAD

**Issue by nvash** [tretyakovsa/Sonoff_WiFi_switch#168]
Скачал вашу прошивку - она замечательная , альтернатив нет :).
Но у меня не получается добавить новый блок на главный экран - не нашел такой возможности. Также не совсем понятно, как вывести показания температуры с ds18b20, и можно ли построить график температуры.
Есть ли эти возможности в полной версии или их нужно доделывать самому? 
При попытке обновиться(прошивка сама предлагает) проходит несколько минут и esp уходит в циклическую перезагрузку.

**Comment by renat2985** [comment]
Добавление новых блоков (редакрирование страниц через CTRL+M) - доступно в полной версии.

Если вы хотите добавить график с тимпературой на главный экран вам необходимо зайди в Конфигурация \ Настройки разработчика \ Если вы работаете с NodeMCU или Wemos, желательно выбрать модуль Manually (там больше возможностей). И ставите галочку на ds18b20 и выбираете нужный вам PIN (менять их не рекомендуется).

![DeepinScreenshot_выберите-область_20190707144552](https://user-images.githubusercontent.com/1230815/60767866-7cd7d880-a0c6-11e9-856c-0562eefc712f.png)

В дизайнах Default и Life вы увидите такие графики:
![DeepinScreenshot_выберите-область_20190707144607](https://user-images.githubusercontent.com/1230815/60767868-8103f600-a0c6-11e9-8f22-db98e571c99e.png)

**Comment by nvash** [comment]
Если датчик не подключен - то графика не будет?

**Comment by renat2985** [comment]
Да, если датчик не подключен - графика не будет. ds18b20 - вы можете подключать параллельно сколько угодно - графиков будет больше.

По поводу обновления, затрудняюсь ответить. Качайте архив с bin файлами и заливайте в свою ESP8266. Вы можете скачать бета (последнею) версию на странице "Конфигурация - Настройки разработчика - Debug". После того как вы установите бета версию на главной странице у вас будет появляться треугольник предлагая поставить снова стабильную версию - ее ставить нельзя, переход снова на стабильную версию мы еще не доделали.
